.travis.yml

```diff
@@ -0,0 +1,26 @@
+
+sudo: required
+dist: xenial
+language: python2.7
+
+services:
+  - docker
+
+branches:
+  only:
+    - master
+
+before_install:
+  - sudo apt-get update
+  - curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
+  - chmod a+x ~/bin/repo
+  - git config --global color.ui false
```

Comment:
All of the above is already done inside the docker image, so no need to do it here.